src/titanium-company-selector.ts

```diff
@@ -92,73 +102,89 @@ export class TitaniumCompanySelectorElement extends PolymerElement {
         {bubbles: true, composed: true, detail: {message: error}}));
   }
 
+  private setCompanyId(id: number|null) {
+    this.dispatchEvent(new CustomEvent(
```

Comment:
when does the actual company id property get set?

Reply:
Oops. good catch.